FSL-DRAFT-MIT.template.md

```diff
@@ -88,15 +88,15 @@ Except for displaying the License Details and identifying us as the origin of
 the Software, you have no right under these Terms and Conditions to use our
 trademarks, trade names, service marks or product names.
 
-### Change License
+## Change License
```

Comment:
This section refers to "the Terms and Conditions above," so it should not itself be a term or condition.